app/controllers/admin/custom_organisations_controller.rb

```diff
@@ -1,11 +1,13 @@
 class Admin::CustomOrganisationsController < AdminController
+  helper_method :sort_column, :sort_direction
+
   def index
-    @custom_organisation_names = CustomOrganisationName.all
+    order_custom_orgs
     @custom_organisation = CustomOrganisationName.new
   end
 
   def create
-    @custom_organisation_names = CustomOrganisationName.all
+    order_custom_orgs
```

Comment:
thinking in terms of readability of the code here:

I feel that it would be good to define all template parameters within the controller, as apposed to within a private method.

```suggestion
    @custom_organisation_names = ordered_custom_orgs
```